<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="liuyingtest" duration-ms="4772" started-at="2017-04-19T12:24:26Z" finished-at="2017-04-19T12:24:31Z">
    <groups>
    </groups>
    <test name="Testlog" duration-ms="4730" started-at="2017-04-19T12:24:26Z" finished-at="2017-04-19T12:24:31Z">
      <class name="com.guangyun.Testlog4j">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.guangyun.Testlog4j@3b6eb2ec]" name="beforeClass" is-config="true" duration-ms="212" started-at="2017-04-19T20:24:26Z" finished-at="2017-04-19T20:24:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.guangyun.Testlog4j@3b6eb2ec]" name="beforeMethod" is-config="true" duration-ms="2885" started-at="2017-04-19T20:24:27Z" finished-at="2017-04-19T20:24:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testSearch()[pri:0, instance:com.guangyun.Testlog4j@3b6eb2ec]" name="testSearch" duration-ms="895" started-at="2017-04-19T20:24:29Z" finished-at="2017-04-19T20:24:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [搜狗搜索引擎 - 上网从搜狗开始]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [搜狗搜索引擎 - 上网从搜狗开始]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.guangyun.Testlog4j.testSearch(Testlog4j.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSearch -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.guangyun.Testlog4j@3b6eb2ec]" name="afterMethod" is-config="true" duration-ms="721" started-at="2017-04-19T20:24:30Z" finished-at="2017-04-19T20:24:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.guangyun.Testlog4j -->
    </test> <!-- Testlog -->
    <test name="TestNGLearn" duration-ms="5" started-at="2017-04-19T12:24:31Z" finished-at="2017-04-19T12:24:31Z">
      <class name="com.guangyun.TestNGLearn">
        <test-method status="PASS" signature="testEquals()[pri:0, instance:com.guangyun.TestNGLearn@6108b2d7]" name="testEquals" duration-ms="1" started-at="2017-04-19T20:24:31Z" finished-at="2017-04-19T20:24:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEquals -->
        <test-method status="FAIL" signature="testSubstract()[pri:0, instance:com.guangyun.TestNGLearn@6108b2d7]" name="testSubstract" duration-ms="1" started-at="2017-04-19T20:24:31Z" finished-at="2017-04-19T20:24:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.guangyun.TestNGLearn.testSubstract(TestNGLearn.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSubstract -->
      </class> <!-- com.guangyun.TestNGLearn -->
    </test> <!-- TestNGLearn -->
    <test name="TestNGLearn1" duration-ms="1" started-at="2017-04-19T12:24:31Z" finished-at="2017-04-19T12:24:31Z">
      <class name="com.guangyun.TestNGLearn1">
        <test-method status="PASS" signature="testAdd()[pri:0, instance:com.guangyun.TestNGLearn1@3498ed]" name="testAdd" duration-ms="1" started-at="2017-04-19T20:24:31Z" finished-at="2017-04-19T20:24:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAdd -->
      </class> <!-- com.guangyun.TestNGLearn1 -->
    </test> <!-- TestNGLearn1 -->
  </suite> <!-- liuyingtest -->
</testng-results>
